ext/odbc/php_odbc_includes.h

```diff
@@ -240,8 +246,8 @@ ZEND_BEGIN_MODULE_GLOBALS(odbc)
     zend_long default_cursortype;
     char laststate[6];
     char lasterrormsg[SQL_MAX_MESSAGE_LENGTH];
-	HashTable *resource_list;
-	HashTable *resource_plist;
+	HashTable results;
```

Comment:
Why do we need to keep results?

Reply:
the list is needed in two places:
- all results are closed when `odbc_close_all()` is called
- when a connection is closed, its related results are closed as well

Reply:
If I see it well, the results are not yet added anywhere to the hash table...